Apps/Encounter/test/unit/Services/EncounterServiceTests.cs

```diff
@@ -151,7 +151,7 @@ public async Task ValidateEncounters(bool canAccessDataSource)
                 GetIConfigurationRoot(),
                 MapperUtil.InitializeAutoMapper());
 
-            RequestResult<IEnumerable<EncounterModel>> actualResult = await service.GetEncounters(hdid);
+            RequestResult<IEnumerable<EncounterModel>> actualResult = await service.GetEncountersAsync(hdid);
```

Comment:
missing token, unless you're considering just relying on the default instance on the method?

Reply:
This is a uniut test.  It is handled by the default.  It is not required.